README.md

```diff
@@ -51,7 +54,18 @@ make
 sudo make check
 sudo make install
 ```
- 
+
+For Raspberry Pi 4
+```
+wget http://www.airspayce.com/mikem/bcm2835/bcm2835-1.62.tar.gz
+tar zxvf bcm2835-1.62.tar.gz
+cd bcm2835-1.62
+./configure
+make
+sudo make check
+sudo make install
+```
+
  Next we are going to download the latest source for PiPROM and compile it. Run the following commands which will move you back into the PiPROM folder and download the latest source code for it.
 
 I added my own line into this. Use mine until the original OP fixes/merges my changes.
```

Comment:
Since these changes have been merged in as part of your PR this part should be removed since it is no longer applicable